src/lib/cache/subCache.cpp

```diff
@@ -757,7 +757,20 @@ void subCacheItemInsert
   //
   // 2. Then the values that have functions/methods for filling/parsing
   //
-  cSubP->throttling = scrP->throttling.parse();
+  if (scrP->throttling.string != "")
+  {
+    //
+    // NOTE: In v1, the throttling of a subscription enters as a string and parsing that string makes sense (right?).
+    //       In v2 however, the throttling is an integer and as the integer value is passed as the sixth parameter of
+    //       this function, the call to scrP->throttling.parse() (the string is empty and parse returns -1) destroys
+    //       the throttling, setting it to -1 (inactive).
+    //       
+    //       For v2, not calling scrP->throttling.parse() is pretty clear.
+    //       What is still a little strange is that v1 has the throttling come is as the sixth parameter but then
```

Comment:
Looking to the code (as in develop, i.e. before this PR):

```
  //
  // Now allocate the subscription
  //
  CachedSubscription* cSubP = new CachedSubscription();
  ...
  cSubP->throttling            = throttling;   // An int64_t, comming from function arguments 
  ...
  cSubP->throttling = scrP->throttling.parse();
```

So it seems that always (i.e. no matter if subscripion came from NGSIv1 and NGSIv2) `scrP->throttling.parse()` is overriding throttling. Thus, probably we could be more "aggrestive" and just remove `cSubP->throttling = scrP->throttling.parse()` (of course, checking that all .test work after that).

Even more aggresive: remove `SubscribeContextRequest*  scrP` from function parameter, replacing it with the actual info that the insert function needs from it (reference string, notifyConditionVector, attributeList and entityIdVector). This is a case in which *too much information* (a throttling-as-a-string, inside de scrP) is causing problems (in particular, overriding another throttling parameter, as number, with the right information).

(Or maybe my reasoning is wrong in some point :))